sled-diagnostics/src/queries.rs

```diff
@@ -254,6 +263,22 @@ pub fn pfiles_process(pid: i32) -> Command {
     cmd
 }
 
+pub fn zfs_list() -> Command {
+    let mut cmd = std::process::Command::new(PFEXEC);
+    cmd.env_clear()
+        .arg(ZFS)
+        .arg("list")
+        .arg("-o")
+        .arg("name,used,avail,quota,reservation,mountpoint");
```

Comment:
we might want to include "mounted" given this past week of debugging

Reply:
Added, thanks!